### PR TITLE
CLI: Manta

### PR DESCRIPTION
**Pull request by stack72**
Adding the stub for a manta cli. This has basic list directory func
only

### CODE REVIEW

File: cmd/config/config.go [hunk 1]
```diff
@@ -19,53 +19,78 @@ type TritonClientConfig struct {
 	Config *triton.ClientConfig
 }
 
-func New() (*TritonClientConfig, error) {
+func buildSSHAgentSigner(keyId string, accountName string) (*authentication.SSHAgentSigner, error) {
```
Comment (sean-) [inline cmd/config/config.go]: func parameter keyId should be keyID

File: cmd/config/config.go [hunk 2]
```diff
@@ -19,53 +19,78 @@ type TritonClientConfig struct {
...
+		KeyID:       keyId,
+		AccountName: accountName,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return signer, nil
+}
+
+func buildPrivateKeySigner(keyId string, accountName string, keyMaterial []byte) (*authentication.PrivateKeySigner, error) {
```
Comment (sean-) [inline cmd/config/config.go]: func parameter keyId should be keyID

File: cmd/config/config.go [hunk 3]
```diff
@@ -93,36 +165,72 @@ func getEnvVar(name string) string {
...
-	keyID := viper.GetString(config.KeySshKeyID)
+	keyID := viper.GetString(config.KeyTritonSshKeyID)
+	if keyID == "" {
+		keyID = getTritonEnvVar("KEY_ID")
+	}
+
+	return keyID
+}
+
+func GetMantaUrl() string {
```
Comment (sean-) [inline cmd/config/config.go]: func GetMantaUrl should be GetMantaURL

File: cmd/internal/config/public.go [hunk 4]
```diff
@@ -9,10 +9,15 @@
 package config
 
 const (
-	KeyAccount        = "general.account"
-	KeyUrl            = "general.url"
-	KeySshKeyMaterial = "general.key-material"
-	KeySshKeyID       = "general.key-id"
+	KeyTritonAccount        = "general.triton.account"
+	KeyTritonUrl            = "general.triton.url"
```
Comment (sean-) [inline cmd/internal/config/public.go]: const KeyTritonUrl should be KeyTritonURL

File: cmd/internal/config/public.go [hunk 5]
```diff
@@ -9,10 +9,15 @@
...
+	KeyTritonSshKeyMaterial = "general.triton.key-material"
```
Comment (sean-) [inline cmd/internal/config/public.go]: const KeyTritonSshKeyMaterial should be KeyTritonSSHKeyMaterial

File: cmd/internal/config/public.go [hunk 6]
```diff
@@ -9,10 +9,15 @@
...
+	KeyTritonSshKeyID       = "general.triton.key-id"
```
Comment (sean-) [inline cmd/internal/config/public.go]: const KeyTritonSshKeyID should be KeyTritonSSHKeyID

File: cmd/internal/config/public.go [hunk 7]
```diff
@@ -9,10 +9,15 @@
...
+
+	KeyMantaAccount        = "general.manta.account"
+	KeyMantaUrl            = "general.manta.url"
```
Comment (sean-) [inline cmd/internal/config/public.go]: const KeyMantaUrl should be KeyMantaURL

File: cmd/internal/config/public.go [hunk 8]
```diff
@@ -9,10 +9,15 @@
...
+	KeyMantaSshKeyMaterial = "general.manta.key-material"
```
Comment (sean-) [inline cmd/internal/config/public.go]: const KeyMantaSshKeyMaterial should be KeyMantaSSHKeyMaterial

File: cmd/internal/config/public.go [hunk 9]
```diff
@@ -9,10 +9,15 @@
...
+	KeyMantaSshKeyID       = "general.manta.key-id"
```
Comment (sean-) [inline cmd/internal/config/public.go]: const KeyMantaSshKeyID should be KeyMantaSSHKeyID

File: cmd/manta/cmd/list/public.go [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+			if err != nil {
+				return err
+			}
+
+			directoryOutput, err := s.GetDirectoryListing(args)
+			if err != nil {
+				return err
+			}
+
+			cons.Write([]byte(fmt.Sprintf("Found %d directory entries", directoryOutput.ResultSetSize)))
```
Comment (sean-) [inline cmd/manta/cmd/list/public.go]: Error return value is not checked

File: cmd/manta/cmd/list/public.go [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+
+			for _, entry := range directoryOutput.Entries {
+				cons.Write([]byte(fmt.Sprintf("\n%s/", entry.Name)))
```
Comment (sean-) [inline cmd/manta/cmd/list/public.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 12]
```diff
@@ -0,0 +1,201 @@
...
+				description = "Use a pager to read the output (defaults to $PAGER, less(1), or more(1))"
+			)
+			var defaultValue bool
+			if isatty.IsTerminal(os.Stdout.Fd()) || isatty.IsCygwinTerminal(os.Stdout.Fd()) {
+				defaultValue = true
+			}
+
+			flags := parent.Cobra.PersistentFlags()
+			flags.BoolP(longName, shortName, defaultValue, description)
+			viper.BindPFlag(key, flags.Lookup(longName))
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 13]
```diff
@@ -0,0 +1,201 @@
...
+				key          = config.KeyLogLevel
+				longOpt      = "log-level"
+				shortOpt     = "l"
+				defaultValue = "INFO"
+				description  = "Change the log level being sent to stdout"
+			)
+
+			flags := parent.Cobra.PersistentFlags()
+			flags.StringP(longOpt, shortOpt, defaultValue, description)
+			viper.BindPFlag(key, flags.Lookup(longOpt))
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 14]
```diff
@@ -0,0 +1,201 @@
...
+				key         = config.KeyLogFormat
+				longOpt     = "log-format"
+				shortOpt    = "F"
+				description = `Specify the log format ("auto", "zerolog", or "human")`
+			)
+			defaultValue := logger.FormatAuto.String()
+
+			flags := parent.Cobra.PersistentFlags()
+			flags.StringP(longOpt, shortOpt, defaultValue, description)
+			viper.BindPFlag(key, flags.Lookup(longOpt))
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 15]
```diff
@@ -0,0 +1,201 @@
...
+				description = "Use ASCII colors"
+			)
+			defaultValue := false
+			if isatty.IsTerminal(os.Stdout.Fd()) || isatty.IsCygwinTerminal(os.Stdout.Fd()) {
+				defaultValue = true
+			}
+
+			flags := parent.Cobra.PersistentFlags()
+			flags.BoolP(longOpt, shortOpt, defaultValue, description)
+			viper.BindPFlag(key, flags.Lookup(longOpt))
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 16]
```diff
@@ -0,0 +1,201 @@
...
+				key          = config.KeyUseUTC
+				longName     = "utc"
+				shortName    = "Z"
+				defaultValue = false
+				description  = "Display times in UTC"
+			)
+
+			flags := parent.Cobra.PersistentFlags()
+			flags.BoolP(longName, shortName, defaultValue, description)
+			viper.BindPFlag(key, flags.Lookup(longName))
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 17]
```diff
@@ -0,0 +1,201 @@
...
+				key          = config.KeyMantaAccount
+				longName     = "account"
+				shortName    = "A"
+				defaultValue = ""
+				description  = "Account (login name). If not specified, the environment variable MANTA_USER will be used"
+			)
+
+			flags := parent.Cobra.PersistentFlags()
+			flags.StringP(longName, shortName, defaultValue, description)
+			viper.BindPFlag(key, flags.Lookup(longName))
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 18]
```diff
@@ -0,0 +1,201 @@
...
+				key          = config.KeyMantaUrl
+				longName     = "url"
+				shortName    = "U"
+				defaultValue = ""
+				description  = "Manta URL. If not specified, the environment variable MANTA_URL will be used"
+			)
+
+			flags := parent.Cobra.PersistentFlags()
+			flags.StringP(longName, shortName, defaultValue, description)
+			viper.BindPFlag(key, flags.Lookup(longName))
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 19]
```diff
@@ -0,0 +1,201 @@
...
+				key          = config.KeyTritonSshKeyID
+				longName     = "key-id"
+				shortName    = "K"
+				defaultValue = ""
+				description  = "This is the fingerprint of the public key matching the key specified in key_path. It can be obtained via the command ssh-keygen -l -E md5 -f /path/to/key. It can be provided via the SDC_KEY_ID or TRITON_KEY_ID environment variables."
+			)
+
+			flags := parent.Cobra.PersistentFlags()
+			flags.StringP(longName, shortName, defaultValue, description)
+			viper.BindPFlag(key, flags.Lookup(longName))
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 20]
```diff
@@ -0,0 +1,201 @@
...
+			const (
+				key          = config.KeyTritonSshKeyMaterial
+				longName     = "key-material"
+				defaultValue = ""
+				description  = "This is the private key of an SSH key associated with the Triton account to be used. If this is not set, the private key corresponding to the fingerprint in key_id must be available via an SSH Agent. It can be provided via the SDC_KEY_MATERIAL or TRITON_KEY_MATERIAL environment variables."
+			)
+
+			flags := parent.Cobra.PersistentFlags()
+			flags.String(longName, defaultValue, description)
+			viper.BindPFlag(key, flags.Lookup(longName))
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 21]
```diff
@@ -0,0 +1,201 @@
...
+		}
+
+		return nil
+	},
+}
+
+func Execute() error {
+
+	rootCmd.Setup(rootCmd)
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 22]
```diff
@@ -0,0 +1,201 @@
...
+
+	conswriter.UsePager(viper.GetBool(config.KeyUsePager))
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 23]
```diff
@@ -0,0 +1,201 @@
...
+
+	if err := logger.Setup(); err != nil {
+		return err
+	}
+
+	for _, cmd := range subCommands {
+		rootCmd.Cobra.AddCommand(cmd.Cobra)
+		cmd.Setup(cmd)
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/version/public.go [hunk 24]
```diff
@@ -0,0 +1,35 @@
...
+
+var Cmd = &command.Command{
+	Cobra: &cobra.Command{
+		Use:          "version",
+		Short:        "print triton cli version",
+		SilenceUsage: true,
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cons := conswriter.GetTerminal()
+			cons.Write([]byte(fmt.Sprintf("Version: %s\n", triton.UserAgent())))
```
Comment (sean-) [inline cmd/manta/cmd/version/public.go]: Error return value is not checked

File: cmd/manta/main.go [hunk 25]
```diff
@@ -0,0 +1,29 @@
...
+
+	"github.com/joyent/triton-go/cmd/manta/cmd"
+	"github.com/rs/zerolog/log"
+	"github.com/sean-/conswriter"
+)
+
+func main() {
+	defer func() {
+		p := conswriter.GetTerminal()
+		p.Wait()
```
Comment (sean-) [inline cmd/manta/main.go]: Error return value is not checked

File: cmd/triton/cmd/root.go [hunk 26]
```diff
@@ -134,7 +134,7 @@ var rootCmd = &command.Command{
 
 		{
 			const (
-				key          = config.KeyUrl
+				key          = config.KeyTritonUrl
```
Comment (cheapRoc) [inline cmd/triton/cmd/root.go]: `KeyTritonURL`

File: cmd/triton/cmd/root.go [hunk 27]
```diff
@@ -148,7 +148,7 @@ var rootCmd = &command.Command{
 
 		{
 			const (
-				key          = config.KeySshKeyID
+				key          = config.KeyTritonSshKeyID
```
Comment (cheapRoc) [inline cmd/triton/cmd/root.go]: `KeyTritonSSHKeyID`

File: cmd/triton/cmd/root.go [hunk 28]
```diff
@@ -162,7 +162,7 @@ var rootCmd = &command.Command{
 
 		{
 			const (
-				key          = config.KeySshKeyMaterial
+				key          = config.KeyTritonSshKeyMaterial
```
Comment (cheapRoc) [inline cmd/triton/cmd/root.go]: `KeyTritonSSHKeyMaterial`

File: cmd/manta/cmd/version/public.go [hunk 29]
```diff
@@ -0,0 +1,35 @@
...
+	triton "github.com/joyent/triton-go"
+	"github.com/joyent/triton-go/cmd/internal/command"
+	"github.com/sean-/conswriter"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &command.Command{
+	Cobra: &cobra.Command{
+		Use:          "version",
+		Short:        "print triton cli version",
```
Comment (kwilczynski) [inline cmd/manta/cmd/version/public.go]: It would be "manta" in this case. :)

File: cmd/internal/config/public.go (hunk 4 above)
Comment (sean-) [inline cmd/internal/config/public.go]: const KeyTritonUrl should be KeyTritonURL

File: cmd/internal/config/public.go (hunk 5 above)
Comment (sean-) [inline cmd/internal/config/public.go]: const KeyTritonSshKeyMaterial should be KeyTritonSSHKeyMaterial

File: cmd/internal/config/public.go (hunk 6 above)
Comment (sean-) [inline cmd/internal/config/public.go]: const KeyTritonSshKeyID should be KeyTritonSSHKeyID

File: cmd/internal/config/public.go (hunk 7 above)
Comment (sean-) [inline cmd/internal/config/public.go]: const KeyMantaUrl should be KeyMantaURL

File: cmd/internal/config/public.go (hunk 8 above)
Comment (sean-) [inline cmd/internal/config/public.go]: const KeyMantaSshKeyMaterial should be KeyMantaSSHKeyMaterial

File: cmd/internal/config/public.go (hunk 9 above)
Comment (sean-) [inline cmd/internal/config/public.go]: const KeyMantaSshKeyID should be KeyMantaSSHKeyID

File: cmd/manta/cmd/list/public.go (hunk 10 above)
Comment (sean-) [inline cmd/manta/cmd/list/public.go]: Error return value is not checked

File: cmd/manta/cmd/list/public.go (hunk 11 above)
Comment (sean-) [inline cmd/manta/cmd/list/public.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 12 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 13 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 14 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 15 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 16 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 17 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 18 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 19 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 20 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 21 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 22 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 23 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/version/public.go (hunk 24 above)
Comment (sean-) [inline cmd/manta/cmd/version/public.go]: Error return value is not checked

File: cmd/manta/main.go (hunk 25 above)
Comment (sean-) [inline cmd/manta/main.go]: Error return value is not checked

File: cmd/manta/cmd/list/public.go (hunk 10 above)
Comment (sean-) [inline cmd/manta/cmd/list/public.go]: Error return value is not checked

File: cmd/manta/cmd/list/public.go (hunk 11 above)
Comment (sean-) [inline cmd/manta/cmd/list/public.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 12 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 13 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 14 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 15 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 16 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 17 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 30]
```diff
@@ -0,0 +1,201 @@
...
+				key          = config.KeyMantaURL
+				longName     = "url"
+				shortName    = "U"
+				defaultValue = ""
+				description  = "Manta URL. If not specified, the environment variable MANTA_URL will be used"
+			)
+
+			flags := parent.Cobra.PersistentFlags()
+			flags.StringP(longName, shortName, defaultValue, description)
+			viper.BindPFlag(key, flags.Lookup(longName))
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 31]
```diff
@@ -0,0 +1,201 @@
...
+				key          = config.KeyTritonSSHKeyID
+				longName     = "key-id"
+				shortName    = "K"
+				defaultValue = ""
+				description  = "This is the fingerprint of the public key matching the key specified in key_path. It can be obtained via the command ssh-keygen -l -E md5 -f /path/to/key. It can be provided via the SDC_KEY_ID or TRITON_KEY_ID environment variables."
+			)
+
+			flags := parent.Cobra.PersistentFlags()
+			flags.StringP(longName, shortName, defaultValue, description)
+			viper.BindPFlag(key, flags.Lookup(longName))
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 32]
```diff
@@ -0,0 +1,201 @@
...
+			const (
+				key          = config.KeyTritonSSHKeyMaterial
+				longName     = "key-material"
+				defaultValue = ""
+				description  = "This is the private key of an SSH key associated with the Triton account to be used. If this is not set, the private key corresponding to the fingerprint in key_id must be available via an SSH Agent. It can be provided via the SDC_KEY_MATERIAL or TRITON_KEY_MATERIAL environment variables."
+			)
+
+			flags := parent.Cobra.PersistentFlags()
+			flags.String(longName, defaultValue, description)
+			viper.BindPFlag(key, flags.Lookup(longName))
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 33]
```diff
@@ -0,0 +1,201 @@
...
+		}
+
+		return nil
+	},
+}
+
+func Execute() error {
+
+	rootCmd.Setup(rootCmd)
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 34]
```diff
@@ -0,0 +1,201 @@
...
+
+	conswriter.UsePager(viper.GetBool(config.KeyUsePager))
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go [hunk 35]
```diff
@@ -0,0 +1,201 @@
...
+
+	if err := logger.Setup(); err != nil {
+		return err
+	}
+
+	for _, cmd := range subCommands {
+		rootCmd.Cobra.AddCommand(cmd.Cobra)
+		cmd.Setup(cmd)
```
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/version/public.go (hunk 24 above)
Comment (sean-) [inline cmd/manta/cmd/version/public.go]: Error return value is not checked

File: cmd/manta/main.go (hunk 25 above)
Comment (sean-) [inline cmd/manta/main.go]: Error return value is not checked

File: cmd/manta/cmd/list/public.go (hunk 10 above)
Comment (sean-) [inline cmd/manta/cmd/list/public.go]: Error return value is not checked

File: cmd/manta/cmd/list/public.go (hunk 11 above)
Comment (sean-) [inline cmd/manta/cmd/list/public.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 12 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 13 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 14 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 15 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 16 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 17 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 30 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 31 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 32 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 33 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 34 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/root.go (hunk 35 above)
Comment (sean-) [inline cmd/manta/cmd/root.go]: Error return value is not checked

File: cmd/manta/cmd/version/public.go [hunk 36]
```diff
@@ -0,0 +1,35 @@
...
+
+var Cmd = &command.Command{
+	Cobra: &cobra.Command{
+		Use:          "version",
+		Short:        "print Joyent manta cli version",
+		SilenceUsage: true,
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cons := conswriter.GetTerminal()
+			cons.Write([]byte(fmt.Sprintf("Version: %s\n", triton.UserAgent())))
```
Comment (sean-) [inline cmd/manta/cmd/version/public.go]: Error return value is not checked

File: cmd/manta/main.go (hunk 25 above)
Comment (sean-) [inline cmd/manta/main.go]: Error return value is not checked

**Comment by kwilczynski** [comment]
For the time being we are fine with all the unchecked errors in Viper. Sometime to address later once the initial client is out.

**Pull request merged**